meteor/sofie-server/data-access/repositories/mongo/mongo-part-repository.ts

```diff
@@ -33,4 +34,22 @@ export class MongoPartRepository extends BaseMongoRepository implements PartRepo
 			})
 		)
 	}
+
+	public async deleteParts(segmentId: string): Promise<void> {
+		this.assertDatabaseConnection(this.deleteParts.name)
+		const parts = await this.getParts(segmentId)
+
+		parts.map((part) => part.id).forEach(async (id) => await this.pieceRepository.deletePieces(id))
+		const partsDeletedResult = await this.getCollection().deleteMany({ segmentId: segmentId })
+
+		// TODO: Figure out how to archive a 'false' acknowledgment, and add test case using that knowledge
```

Comment:
Are you planning on doing this TODO in this task or?

Reply:
Would love to, but i am not sure how i could get a `false` acknowledgment to be returned.

I did read somewhere that passing an `undefined` value to the `_id` that is being searched for would do it, but that is not a valid argument for the method. Because of that, i currently do not know how to get into this state.

If you have any suggestions as to how to do it, or if you simply want me to remove that check, do let me know.

Reply:
You have written a TODO so I am interested in knowing when you are planning to do the TODO. So we don't end up having a lot of TODOs in the code that just ends up staying there.

If you don't plan on doing the TODO, then the TODO should be deleted.

If you plan on doing it, then your task isn't done until you have done ;)

Reply:
I have now a couble of times since writing the TODO, tried to figure out when a false `acknowledgment` will occur during a delete.

There seems to be plenty of examples for it happening during update or insert - just not delete...

Should the check be removed, or how do you want me to handle it?

Reply:
This seems to be quite specific to the chosen implementation. Your tests should ideally "just" tests the required functionality.
So if you have  testscases for:
- Entity is there - correct id - it's deleted
- Entitiy is there - wrong id - it's not delete/handled how we want to handle it
- Entity isn't there - handled how we want to handle it
- Whatever tests you need to assert we are also calling the "next" repository
Then I think you are covered for now and could delete the TODO :)

Reply:
The test cases you listed do exist, so i have removed the todo comment. 
The inside of the `if` is not hit by any test, but i guess that is fine...